homeassistant/components/spotify/config_flow.py

```diff
@@ -34,6 +34,14 @@ def extra_authorize_data(self) -> dict:
             "user-read-playback-state",
             # Needed to determine if the user has Spotify Premium
             "user-read-private",
+            # Needed for media browsing
```

Comment:
Does this mean users need to reauthenticate ? Is this case already handled ?

Reply:
Yes. No, not handled yet.

Reply:
For now warning will be logged when there is a mismatch in scope.

Reply:
That sounds weird. Can't we simply bump the integration entry version? If it doesn't match, re-trigger auth?

Reply:
Added PR to this branch to implement re-authentication to correct the scopes:

<https://github.com/cgtobi/home-assistant/pull/5>